webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -386,3 +387,41 @@ def show_filesize(bytes, unit='bytes'):
 @library.global_function
 def booleanish_to_boolean(value):
     return str(value).lower() in ('1', 'true', 'yes')
+
+
+@library.global_function
+def dict_as_html_tree(structure):
+    if not isinstance(structure, dict):
+        # then it has to be a JSON string
+        structure = json.loads(structure)
+
+    def tree(container, depth=1):
+        P = ' ' * 4
+        html = []
+
+        # The reason for using `key=lambda s: s.lower()` instead
+        # of just `str.lower` is because it's not a guarantee that
+        # all strings are byte strings. Or unicode for that matter.
```

Comment:
I don't understand this comment at all. What're the two things you're comparing here?